apps/wallets/aa/nextjs/package.json

```diff
@@ -15,7 +15,8 @@
         "ethers": "5.7.2",
         "next": "13.5.6",
         "react": "^18",
-        "react-dom": "^18"
+        "react-dom": "^18",
+        "viem": "2.13.1"
```

Comment:
pnpm again being awesome, discovering incorrectly used dependencies

we were importing and using viem, but its not set as a dep - now it is